docker-compose.yml

```diff
@@ -7,23 +7,36 @@ services:
     volumes:
       - imaginethis-db-vol:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 5432
 
   imaginethis-backend:
     build: ./ImagineThisServer
     container_name: imaginethis-backend
     environment:
-      - DB_SERVER=imaginethis-db
+      - IMAGINE_THIS_DB_HOST=postgresql://imaginethis-db
+      - IMAGINE_THIS_DB_PORT=5432
+      - IMAGINE_THIS_DB_NAME=
+      - IMAGINE_THIS_USERNAME=
+      - IMAGINE_THIS_PASSWORD=
+      - CORS_ORIGIN=
+      - SPRING_PROFILES_ACTIVE=prod
+      # absolute location to store generated code
+      - PWD=
```

Comment:
This doesn't need to be defined, that's a system defined environment variable (like `pwd` command). I'm using it in back-end so that the output storage goes to the directory where the Java backend is running.